deepmd/descriptor/se_a_mask.py

```diff
@@ -0,0 +1,429 @@
+        self.sub_sess = tf.Session(graph = sub_graph, config=default_tf_session_config)
+        self.original_sel = None
+
+
+    def compute_input_stats (self,
+                             data_coord : list, 
+                             data_box : list, 
+                             data_atype : list, 
+                             natoms_vec : list,
+                             mesh : list, 
+                             input_dict : dict
+    ) -> None :
+        """
+        Compute the statisitcs (avg and std) of the training data. The input will be normalized by the statistics.
+        
+        Parameters
+        ----------
+        data_coord
+                The coordinates. Can be generated by deepmd.model.make_stat_input
+        data_box
+                The box. Can be generated by deepmd.model.make_stat_input
+        data_atype
+                The atom types. Can be generated by deepmd.model.make_stat_input
+        natoms_vec
+                The vector for the number of atoms of the system and different types of atoms. Can be generated by deepmd.model.make_stat_input
+        mesh
+                The mesh for neighbor searching. Can be generated by deepmd.model.make_stat_input
+        input_dict
+                Dictionary for additional input
+        """
+        mask_matrix = input_dict['mask_matrix']
+        all_davg = []
+        all_dstd = []
+        if True:
+            sumr = []
+            suma = []
+            sumn = []
+            sumr2 = []
+            suma2 = []
+            for cc,tt,mm,nn in zip(data_coord,data_atype,mask_matrix, natoms_vec) :
+                sysr,sysr2,sysa,sysa2,sysn \
+                    = self._compute_dstats_sys_smth(cc,tt,mm, nn)
+                sumr.append(sysr)
+                suma.append(sysa)
+                sumn.append(sysn)
+                sumr2.append(sysr2)
+                suma2.append(sysa2)
+            sumr = np.sum(sumr, axis = 0)
+            suma = np.sum(suma, axis = 0)
+            sumn = np.sum(sumn, axis = 0)
+            sumr2 = np.sum(sumr2, axis = 0)
+            suma2 = np.sum(suma2, axis = 0)
+            for type_i in range(self.ntypes) :
+                davgunit = [sumr[type_i]/(sumn[type_i]+1e-15), 0, 0, 0]
+                dstdunit = [self._compute_std(sumr2[type_i], sumr[type_i], sumn[type_i]), 
+                            self._compute_std(suma2[type_i], suma[type_i], sumn[type_i]), 
+                            self._compute_std(suma2[type_i], suma[type_i], sumn[type_i]), 
+                            self._compute_std(suma2[type_i], suma[type_i], sumn[type_i])
+                            ]
+                
+                davg = np.tile(davgunit, self.ndescrpt // 4)
+                dstd = np.tile(dstdunit, self.ndescrpt // 4)
+                all_davg.append(davg)
+                all_dstd.append(dstd)
+
+        if not self.set_davg_zero:
+            self.davg = np.array(all_davg)
+        self.dstd = np.array(all_dstd)
```

Comment:
Correct me if I am wrong, but it looks like `descrpt_se_a_mask` never uses `avg` and `std`? Is it an expected behavior or something unimplemented?
The computing of `avg` and `std` looks useless.

Reply:
Note that compression relies on the correct avg and std. If not used, please set to 0 and 1.

https://github.com/deepmodeling/deepmd-kit/blob/e8a57caa127d3a6c7d262c3e2567a634a0c17259/deepmd/utils/tabulate.py#L397-L399

Reply:
Yes, for this version, it never uses `avg` and `std`. But I don't know that if any other functions, such as the methods in `se_a` or outside functions, will call this `avg` or `std`. This implementation is used for the compatibility of `DescrptSeAMask` with other ops.

Reply:
> Note that compression relies on the correct avg and std. If not used, please set to 0 and 1.
> 
> https://github.com/deepmodeling/deepmd-kit/blob/e8a57caa127d3a6c7d262c3e2567a634a0c17259/deepmd/utils/tabulate.py#L397-L399

OK, it will be fixed.